src/main/java/com/google/devtools/build/lib/bazel/bzlmod/RegistryFunction.java

```diff
@@ -64,8 +67,13 @@ public SkyValue compute(SkyKey skyKey, Environment env)
       RegistryFunction.LAST_INVALIDATION.get(env);
     }
 
-    BazelLockFileValue lockfile = (BazelLockFileValue) env.getValue(BazelLockFileValue.KEY);
-    if (lockfile == null) {
+    var lockfiles =
+        env.getValuesAndExceptions(
+            ImmutableList.of(BazelLockFileValue.KEY, BazelLockFileValue.HIDDEN_KEY));
+    BazelLockFileValue lockfile = (BazelLockFileValue) lockfiles.get(BazelLockFileValue.KEY);
+    BazelLockFileValue persistentLockfile =
```

Comment:
rename to `hiddenLockfile`

Reply:
hmm, since we're not storing registry file hashes in the hidden lockfile right now, the changes in this file are unnecessary, right?